tox.ini

```diff
@@ -0,0 +1,18 @@
+; this is a tox config for running tests
+; under all supported Python interpreters
+
+[tox]
+envlist = py27,py34,py35,py36
+
+[testenv]
+setenv =
+    KAFKA_BROKER={env:KAFKA_BROKER:}
+    KAFKA_BIN={env:KAFKA_BIN:}
+    ZOOKEEPER={env:ZOOKEEPER:}
```

Comment:
Can you use [passenv](http://tox.readthedocs.io/en/latest/config.html#confval-passenv=SPACE-SEPARATED-GLOBNAMES) here?

Reply:
hey @kmike I couldn't make it work with travis and passenv, maybe was doing something wrong, but I've moved all the environment definitions to tox.ini because the env. are always the same.

Reply:
I think the problem was that you were setting env variable inside a shell script, and it was local to this script. 

Env variables are always the same on Travis, but you can run tox locally, and this is where passing custom kafka location may come handy. What do you think about moving these constants to travis.yml, so that users who run tox.ini locally can override them?

Reply:
I see, great then, couldn't get it to work before, probably because they weren't global to the matrix (I've just realized that), I agree that using passenv should be better